azure-pipelines.yml

```diff
@@ -0,0 +1,182 @@
+  displayName: Verify code signing
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- task: VSTest@2
+  inputs:
+    testFiltercriteria: TestCategory!=FailsInCloudTest
+    searchFolder: $(System.DefaultWorkingDirectory)\bin\
+    testAssemblyVer2: |
+      $(BuildConfiguration)\**\*test*.dll
+      !**\obj\**
+    platform: $(BuildPlatform)
+    configuration: $(BuildConfiguration)
+  displayName: Run Tests
+  condition: and(succeeded(), ne(variables['SignType'], 'real'))
+
+- task: MicroBuildCleanup@1
+  displayName: MicroBuild Cleanup
+  condition: succeededOrFailed()
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    PathtoPublish: $(Build.ArtifactStagingDirectory)/MicroBuild/ESRPClient
+    ArtifactName: esrpclient_logs
+    ArtifactType: Container
+  displayName: 'Publish Artifact: ESRP signing logs'
+  condition: eq(variables['SignType'], 'real')
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    PathtoPublish: $(Build.ArtifactStagingDirectory)/build_logs
+    ArtifactName: build_logs
+    ArtifactType: Container
+  displayName: 'Publish Artifact: build logs'
+  condition: succeededOrFailed()
+
+## Following steps are skipped in PR builds
+
+- task: CopyFiles@1
+  displayName: Collecting symbols artifacts
+  inputs:
+    SourceFolder: bin/$(BuildConfiguration)/net472
+    Contents: |
+      **/Microsoft.VisualStudio.Jdt?(*.dll|*.pdb|*.xml)
+      !**/*Test*
+    TargetFolder: $(Build.ArtifactStagingDirectory)/symbols
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- task: PublishSymbols@2
+  inputs:
+    SymbolsFolder: $(Build.ArtifactStagingDirectory)/symbols
+    SearchPattern: '**\*.pdb'
+    SymbolServerType: TeamServices
+    ArtifactServices.Symbol.AccountName: microsoft
+    ArtifactServices.Symbol.PAT: $(System.AccessToken)
+    ArtifactServices.Symbol.UseAAD: false
+  displayName: Archive symbols to VSTS
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['SignType'], 'real'))
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)/symbols'
+    ArtifactName: symbols
+    publishLocation: Container
+  displayName: 'Publish Artifact: symbols'
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- task: CopyFiles@1
+  displayName: Collecting packages
+  inputs:
+    SourceFolder: bin/$(BuildConfiguration)/packages
+    Contents: |
+      *.nupkg
+    TargetFolder: $(Build.ArtifactStagingDirectory)/packages
+    flattenFolders: false
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- task: PublishBuildArtifacts@1
+  displayName: Publish packages
+  inputs:
+    PathtoPublish: $(Build.ArtifactStagingDirectory)/packages
+    ArtifactName: packages
+    ArtifactType: Container
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- task: NuGetCommand@2
```

Comment:
I don't think this is needed (publishing to VS feeds). Could you check if those feeds contain older versions of the packages?
On the other hand, I don't think that causes any side effects

Reply:
from what I can tell the feed already has some versions out from 2017.
![image](https://user-images.githubusercontent.com/62913657/215172563-dc961e64-4fdc-4da7-81db-831775e4f16a.png)